### PR TITLE
Improve code coverage of zookeeper config center

### PR DESCRIPTION
**Pull request by Patrick0308**
Improve code coverage of zookeeper config center

relates #518

### CODE REVIEW

File: config_center/zookeeper/impl.go [hunk 1]
```diff
@@ -81,32 +79,6 @@ func newZookeeperDynamicConfiguration(url *common.URL) (*zookeeperDynamicConfigu
...
-	c := &zookeeperDynamicConfiguration{
-		url:      url,
-		rootPath: "/" + url.GetParam(constant.CONFIG_NAMESPACE_KEY, config_center.DEFAULT_GROUP) + "/config",
-	}
-	var (
-		tc  *zk.TestCluster
-		err error
-	)
-	tc, c.client, _, err = zookeeper.NewMockZookeeperClient("test", 15*time.Second, opts...)
-	if err != nil {
```
Comment (hxmhlt) [inline config_center/zookeeper/impl.go:94]: zookeeper.NewMockZookeeperClient may delete?

File: config_center/zookeeper/impl.go [hunk 2]
```diff
@@ -81,32 +79,6 @@ func newZookeeperDynamicConfiguration(url *common.URL) (*zookeeperDynamicConfigu
...
-		logger.Errorf("mock zookeeper client start error ,error message is %v", err)
-		return tc, c, err
-	}
-	c.wg.Add(1)
-	go zookeeper.HandleClientRestart(c)
-
```
Comment (hxmhlt) [inline config_center/zookeeper/impl.go:100]: Is zookeeper.HandleClientRestart(c)  coveraged ?

File: config_center/zookeeper/impl.go (hunk 1 above)
Comment (Patrick0308) [reply config_center/zookeeper/impl.go:94]: `zookeeper.NewMockZookeeperClient` be used in other place.

File: config_center/zookeeper/impl.go (hunk 2 above)
Comment (Patrick0308) [reply config_center/zookeeper/impl.go:100]: Yeah, it's be covered. I don't know why this coverage be smaller. In my computer, `cluster/cluster_impl/base_cluster_invoker.go` coverage is bigger than 80%.

**Pull request merged**